src/util.py

```diff
@@ -13,6 +13,11 @@
              'db_host': 'localhost',
              'db_name': 'psd_project'}
 
+# remote database to log in
+db_config_remote = {'user': 'zguo4',
+                    'password': 'SJk+6L4K3fKX',
+                    'db_host': 'db1.mcs.slu.edu',  # 'db1.mcs.slu.edu',
```

Comment:
Is this correct for connecting to hopper?

Reply:
I get the method of connecting to hopper. We should distribute the programs on hopper then DB would work